app/content/blog/wiki/allagan-tools-inventory-analysis.md

```
@@ -0,0 +1,93 @@
+Our Allagan Data tool will analyze your inventory and items you are selling on the marketboard.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)